src/List.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class List extends React.PureComponent {
+  state = {
+    todo: this.props.todo,
+    users: this.props.user,
+    checkboxStatus: false,
+  }
+
+  handlerCB = () => {
```

Comment:
how would I understand what CB means? make the name clear and understandable for everybody

Reply:
fixed